cmd/esc/cli/testdata/env-tag.yaml

```diff
@@ -0,0 +1,34 @@
+run: |
+  esc env tag ls test-org/env
+  esc env tag test-org/env owner pulumi
+  esc env tag get test-org/env owner
```

Comment:
do we need a separate `get` command? could `tag` w/o a value just return the current value of the tag?

Reply:
It's not required here since the return value of `tag` does that but I added it to the test to demonstrate that it is being persisted